Apollonius_graph_2/include/CGAL/Parabola_2.h

```diff
@@ -59,7 +59,7 @@ class Parabola_2
 #else
   static const FT& STEP()
   {
-    static FT step_(2);
+    static const FT step_(2);
```

Comment:
I think I have already warned about the non-POD `static const` variables: their initialization cannot be done at compile time, in particular if `FT` is a `Lazy_exact_nt` that allocates memory, and thus that initialization is not thread-safe: it is done by the "first" thread that enters the function, and there can be two concurrent threads trying to initialize the variable at the same time. In this case, the effects of that race-condition will probably only be memory leaks, but in general that could be more problematic.